dataflow/src/main/java/org/checkerframework/dataflow/cfg/playground/LiveVariablePlayground.java

```diff
@@ -18,14 +18,14 @@
     public static void main(String[] args) {
 
         /* Configuration: change as appropriate */
-        String inputFile = "Test.java"; // input file name and path
-        String outputDir = "cfg"; // output directory
+        String inputFile = "dataflow/tests/live-variable/Test.java"; // input file name and path
```

Comment:
Is adding the directory necessary here and on the next line? I think the playground is supposed to be used in an arbitrary directory and then looks for `Test.java` in that directory.

Reply:
Sorry, that's a mistake....